scripts/bash/run_enceladus.sh

```diff
@@ -231,6 +257,13 @@ add_to_cmd_line() {
     fi
 }
 
+# Puts Spark configuration properties to the command line
+add_spark_conf_cmd() {
```

Comment:
In some `ifs` you use the single bracket in some you use double ones. There is a difference between them. [Bash manual for more](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Conditional-Constructs)

Reply:
Yeah, I usually go with single brackets is it is sufficient for the task (since it is 'more portable'), and go for double ones when what I want is easier to achieve that way.

Would you prefer it to be consistent across our scripts? If yes, I'd use double ones.
Maybe in different PR...